tests/legacy/test_jax_old.py

```diff
@@ -508,8 +477,9 @@ def cost(a, b):
         expected = -2 * np.sin(2 * a)
         assert np.allclose(jac, expected, atol=tol, rtol=0)
 
-    def test_jit_grad_with_backward_grad_on_execution(self, execute_kwargs, interface):
-        """Test jax jit grad for adjoint diff method in backward grad_on_execution"""
+    @pytest.mark.xfail
+    def test_jit_grad_with_backward_mode(self, execute_kwargs, interface):
+        """Test jax jit grad for adjoint diff method in backward mode"""
```

Comment:
Is this something that we're supposed to support?

Reply:
It is in the old interface (old return type) I talk with Tom and we decided it was not worth updating as it should go away in two releases.